tests/v2/validation/provisioning/hosted/aks/hosted_provisioning_test.go

```diff
@@ -129,7 +129,7 @@ func (h *HostedAKSClusterProvisioningTestSuite) testProvisioningHostedAKSCluster
 	require.NoError(h.T(), err)
 	assert.NotEmpty(h.T(), clusterToken)
 
-	err = nodestat.IsNodeReady(rancherClient, clusterResp.ID)
+	err = nodestat.AllRKE1NodeReady(rancherClient, clusterResp.ID)
```

Comment:
I know we renamed this RKE1, but why? If hosted clusters use the same function it's a little confusing.

Reply:
maybe I could update to `AllManagementNodeReady`?